Resources/Prototypes/Entities/Structures/Furniture/beds.yml

```
@@ -81,6 +81,18 @@
   - type: Construction
     graph: bed
     node: medicalbed
+    #Far Horizons-Start - Letting Medical beds also receive the operating computer visuals
+  - type: OperatingTable
+  - type: InteractionOutline
+  - type: DeviceList
+  - type: DeviceNetwork
+    deviceNetId: Wired
+  - type: DeviceLinkSink
+    ports:
+      - VitalsMonitorReceiver
```

```suggestion
      - BodyScannerReceiver
```

The monitor is using `BodyScannerSender` and while it technically works with missmatched links - I'd rather keep it simple

If you want to use the separate Receiver for names - add it to default links in `BodyScannerSender`

---

The vitals monitor receiver is just for the device linking name essentially.

The regular operating table I stole components from has the device connection showing Operating Table instead of Medical Bed

---

In that case please also notice missing LOC strings for sender.

Also make it so sender has default links to both body scanner receiver (when connecting it to surgery table) and vitals monitor receiver

---

The operating table has not been changed. It still connects to the operating console.